store/sagas/contacts.sagas.js

```diff
@@ -22,141 +21,181 @@ export function* getAll({ domain, token }) {
   yield put({
     type: 'REQUEST',
     payload: {
-      url: `https://${user.domain}/wp-json/dt/v1/contact/${urlPart}`,
+      url: `https://${domain}/wp-json/dt-posts/v2/contacts/${contactId}`,
       data: {
-        method: 'POST',
+        method: 'GET',
         headers: {
           'Content-Type': 'application/json',
-          Authorization: `Bearer ${user.token}`,
+          Authorization: `Bearer ${token}`,
         },
-        body: JSON.stringify({
-          title: `${contact.name}`,
-          // contact_phone: [{ value: `${contact.contact_phone}` }, { value: "999-99-9999" }],
-          // contact_phone: [{ value: `${contact.contact_phone}` }],
-          // contact_email: [{ value: `${contact.contact_email}` }],
-          // sources: { values: [{ value: "linkedin" }, { value: "referral" }]},
-          // locations: { values: [{ value: "36" }, { value: 35 }]}
-        }),
       },
-      action: actions.CONTACTS_SAVECONTACT_RESPONSE,
+      action: actions.CONTACTS_GETBYID_RESPONSE,
     },
   });
 
-  // handle response
   try {
-    // TODO: will this block and cause responses to be missed?
-    // use channel instead
-    const res = yield take(actions.CONTACTS_SAVECONTACT_RESPONSE);
+    const res = yield take(actions.CONTACTS_GETBYID_RESPONSE);
     if (res) {
       const response = res.payload;
-      // console.log('*** SAVE CONTACT RESPONSE ***', response);
       const jsonData = yield response.json();
       if (response.status === 200) {
-        // NOTE: create contact endpoint only returns 'post_id' and 'permalink'
-        const key = jsonData.post_id ? jsonData.post_id.toString() : jsonData.ID.toString();
-        // refresh state w/ D.T key/ID for new contact, or to sync existing
-        yield put({ type: actions.CONTACTS_SAVECONTACT_SUCCESS, contact, key });
-        if (contact.initial_comment) {
-          // submit the comment now, AFTER receiving the D.T key/ID
-          delayPostInitialComment(user, contact.initial_comment, key);
-        }
+        const contact = jsonData;
+        yield put({
+          type: actions.CONTACTS_GETBYID_SUCCESS,
+          contact: {},
+        });
       } else {
         yield put({
-          type: actions.CONTACTS_SAVECONTACT_FAILURE,
+          type: actions.CONTACTS_GETBYID_FAILURE,
           error: {
             code: jsonData.code,
             message: jsonData.message,
+            data: jsonData.data,
           },
         });
       }
     }
   } catch (error) {
-    // console.log('*** SAVE CONTACT ERROR ***', error);
-    yield put({ type: actions.CONTACTS_SAVECONTACT_FAILURE, error: { code: '400', message: '(400) Unable to process the request. Please try again later.' } });
+    console.log('getById() ->error');
+    console.log(error);
+    yield put({
+      type: actions.CONTACTS_GETBYID_FAILURE,
+      error: {
+        code: '400',
+        message: '(400) Unable to process the request. Please try again later.',
```

Comment:
As previous comment, can we utilize the `error` parameter of the catch statement?